README.md

```diff
@@ -1,22 +1,31 @@
-# D3b Bixu Repository Template
+# D3b DGD Collaboration
 
-Use this template to bootstrap a new D3b bixu repository 
+Hosts various code and workflows of mutual interest to D3b and DGD 
```

Comment:
Oh, well, I guess I can make that change. I was sort of cheating and making it more DGD-specific since that was the main customer. I had hid some related options, so I can bring that back in